markdown/blog/2024-marketing-report.md

```diff
@@ -0,0 +1,187 @@
+
+Here are some of the benefits of using the Buffer subscription
+- **Centralized Dashboard:** Manage all social media platforms from a single interface, making tracking and organizing content easier. 
+- **Improved Security:** No need for multiple people to log into the actual social media accounts. This minimizes the risks of unauthorized changes or breaches.
+- **Better Analytics and Insights:** Access in-depth analytics to understand which posts perform and engagement metrics best and adjust strategies accordingly. 
+- **Content Approval Workflow:** Allow team members to draft posts for review, ensuring high-quality content that aligns with AsyncAPI’s voice.
+
+
+Please note that I currently do not have direct access to AsyncAPI’s social media accounts. Buffer provides an effective solution for posting content, monitoring performance, and maintaining the high standards of AsyncAPI's public presence without broad access.
+
+As part of the benefit of using Buffer, we have access to our social media analytics. Below are the analytics for LinkedIn and X (formally Twitter) from May 20th to December 10th, 2024.
+
+
+![AsyncAPI LinkedIn engagement metrics showing follower growth and post metrics from Buffer analytics](/img/posts/marketing-images/linkedIn-metrics-insights.webp)
+
+
+![AsyncAPI LinkedIn engagement metrics showing follower growth and post performance from Buffer analytics](/img/posts/marketing-images/linkedin-performance-statistics.webp)
+
+
+The images above show that LinkedIn's performance is great. Here’s a breakdown of our LinkedIn metrics from Buffer’s analysis:
+
+- **Followers:** We’ve grown to 3,905 followers, marking a 19% increase in our audience size. 
+
+- **Impressions:** Our posts were seen 68,440 times, a remarkable 120% growth, indicating broader visibility. 
+
+- **New Followers:** We gained 620 new followers, reflecting a 119% increase in audience acquisition. 
+
+- **Clicks:** Our content drove 3,213 clicks, an impressive 205% growth, showcasing heightened interest and engagement.
+
+- **Posts:** We published 176 posts, up by 209%, emphasizing consistent and active content sharing. 
+
+- **Engagement Rate:** Our engagement rate reached 5.83%, improving by 31%, indicating stronger interactions with our audience. 
+
+Our LinkedIn efforts are yielding fantastic results, with significant growth in visibility, engagement, and audience acquisition. The impressive metrics reflect the value and relevance of our content. However, we're still up for improvement.
+
+![AsyncAPI Twitter (X) engagement metrics showing follower growth and tweet metrics from Buffer analytics](/img/posts/marketing-images/x-metrics-insights.webp)
+
+![AsyncAPI Twitter (X) engagement metrics showing follower growth and tweet performance from Buffer analytics](/img/posts/marketing-images/x-performance-statistics.webp)
```

Comment:
I'm not sure we should share that numbers, they are kinda weird and not realistic - if you look closely at percentage, it looks like in the same period in previous year we did not post a single thing. In buffer when you see the diagram, you have a slider "previous period" and then you turn it on, line showing previous period shows 0 all the time - not possible